nntrainer/tensor/tensor.cpp

```diff
@@ -43,6 +43,10 @@
 #include <tensor.h>
 #include <util_func.h>
 
+#ifdef _WIN32
+#define uint unsigned int
```

Comment:
Use typedef instead. and try to define once and refer to a common header. You are redefining and it will become problem a few months later.